services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/entity/decider/EntityDeciderFunctions.java

```diff
@@ -175,10 +162,16 @@
 	DENY_IF_HAS_NOT_ACCEPTED_TERMS_OF_USE((c) -> {
 		if (!c.getUser().acceptsTermsOfUse()) {
 			return Optional.of(new UsersEntityAccessInfo(c,
-					AuthorizationStatus.accessDenied(YOU_HAVE_NOT_YET_AGREED_TO_THE_SYNAPSE_TERMS_OF_USE)));
+					AuthorizationStatus.accessDenied(ERR_MSG_YOU_HAVE_NOT_YET_AGREED_TO_THE_SYNAPSE_TERMS_OF_USE)));
 		} else {
 			return Optional.empty();
 		}
+	}),
+	/**
+	 * Unconditional deny.
+	 */
+	DENY((c) -> {
```

Comment:
We might want to add the ACCESS_TYPE to the context so that the default deny message makes more sense.